microcosm_caching/decorators.py

```diff
@@ -20,7 +21,15 @@ def get_metrics(graph):
         return None
 
 
-def cache_key(cache_prefix, key):
+def cache_key(cache_prefix, schema, args, kwargs) -> str:
+    """
+    Hash a key according to the schema and input args.
+
+    """
+    key = (schema.__name__,) + args
```

Comment:
Yes, technically args are being included here. I can always remove if it's confusing.